### PR TITLE
ES-865324 - Added Github sample for the Knowledge Base

### PR DESCRIPTION
**Pull request by VijayadharshiniMathiyalagan**
Description:
Added Sample for the below:

1. Apply-indentation-for-import-content - Paragraphs
2. Remove-empty-column-after-mail-merge - Mail-merge
3. Replace-Excel-embedded-image-to-PNG - Word_to_PDF

### CODE REVIEW

File: Paragraphs/Apply-indentation-for-import-content/.NET/Apply-indentation-for-import-content/Data/Temporary.docx
Comment (LokeshBaskar) [inline Paragraphs/Apply-indentation-for-import-content/.NET/Apply-indentation-for-import-content/Data/Temporary.docx:1]: Remove unused file

**Review by LokeshBaskar — CHANGES_REQUESTED** [review]

File: Word-to-PDF-Conversion/Replace-Excel-embedded-image-to-PNG/.NET/Replace-Excel-embedded-image-to-PNG/Program.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+// Load the Word document into a WordDocument instance
+using var tempDocument = new WordDocument(inputStream, FormatType.Automatic);
+// Call a method to replace embedded Excel objects in the document with images
+ReplaceExcelToImage(tempDocument);
+// Convert the Word document to a PDF using the DocIORenderer component
+using PdfDocument pdf = docIORenderer.ConvertToPDF(tempDocument);
+// Create a file stream to save the output PDF document
+FileStream outputStream = new FileStream(Path.GetFullPath(@"Output/Output.pdf"), FileMode.Create, FileAccess.Write);
+// Save the generated PDF to the specified file stream
+pdf.Save(outputStream);
```
Comment (LokeshBaskar) [inline Word-to-PDF-Conversion/Replace-Excel-embedded-image-to-PNG/.NET/Replace-Excel-embedded-image-to-PNG/Program.cs:33]: Either dispose or use using statement

File: Word-to-PDF-Conversion/Replace-Excel-embedded-image-to-PNG/.NET/Replace-Excel-embedded-image-to-PNG/Program.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+
+void ReplaceExcelToImage(WordDocument wordDocument)
```
Comment (LokeshBaskar) [inline Word-to-PDF-Conversion/Replace-Excel-embedded-image-to-PNG/.NET/Replace-Excel-embedded-image-to-PNG/Program.cs:36]: XML comments

**Review by LokeshBaskar — CHANGES_REQUESTED** [review]
Please address the feedback.

File: Word-to-PDF-Conversion/Replace-Excel-embedded-image-to-PNG/.NET/Replace-Excel-embedded-image-to-PNG/Program.cs (hunk 1 above)
Comment (VijayadharshiniMathiyalagan) [reply Word-to-PDF-Conversion/Replace-Excel-embedded-image-to-PNG/.NET/Replace-Excel-embedded-image-to-PNG/Program.cs:26]: addressed

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**